lib/install.js

```diff
@@ -313,75 +313,44 @@ function install (gyp, argv, callback) {
 
       function downloadNodeLib (done) {
         log.verbose('on Windows; need to download `' + release.name + '.lib`...')
-        var dir32 = path.resolve(devDir, 'ia32')
-          , dir64 = path.resolve(devDir, 'x64')
-          , libPath32 = path.resolve(dir32, release.name + '.lib')
-          , libPath64 = path.resolve(dir64, release.name + '.lib')
-
-        log.verbose('32-bit ' + release.name + '.lib dir', dir32)
-        log.verbose('64-bit ' + release.name + '.lib dir', dir64)
-        log.verbose('`' + release.name + '.lib` 32-bit url', release.libUrl32)
-        log.verbose('`' + release.name + '.lib` 64-bit url', release.libUrl64)
-
-        var async = 2
-        mkdir(dir32, function (err) {
-          if (err) return done(err)
-          log.verbose('streaming 32-bit ' + release.name + '.lib to:', libPath32)
-
-          try {
-            var req = download(gyp, process.env, release.libUrl32, cb)
-          } catch (e) {
-            return cb(e)
-          }
-
-          req.on('error', done)
-          req.on('response', function (res) {
-            if (res.statusCode !== 200) {
-              done(new Error(res.statusCode + ' status code downloading 32-bit ' + release.name + '.lib'))
-              return
+        var archs = ['ia32', 'x64']
+        var async = archs.length
```

Comment:
There are many variables named `async` in the code above,  I just used the same name.